eZ/Bundle/EzPublishElasticsearchBundle/DependencyInjection/Compiler/HttpClientPass.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * This file is part of the eZ Publish Kernel package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace eZ\Bundle\EzPublishElasticsearchBundle\DependencyInjection\Compiler;
+
+/**
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/commit/5bd8b3e78eb03bd0372dbc38da36074c34223314.